types.d.ts

```diff
@@ -3089,6 +3089,11 @@ export declare class ConformerGenerator {
    * @returns - This conformer's contribution to all conformers.
    */
   getPreviousConformerContribution(): number;
+
+  /** 
+   * Returns an iterator of molecule conformers.
+   */
+  [Symbol.iterator](): IterableIterator<Molecule>;
```

Comment:
Should we put it under a method like `conformerGenerator.conformersAsMolecule()` ? For consistency if we add the `Conformer` class. There can only be one direct iterator on the generator.

In for of, this currently looks like:
```js
for (const molecule of conformerGenerator) {}
```

If we change, it would be:
```js
for (const molecule of conformerGenerator.conformersAsMolecule()) {}
```